test/plan-marshall/build-gradle/test_gradle_find_project_command.py

```
@@ -0,0 +1,206 @@
+# =============================================================================
+# project_path lookup
+# =============================================================================
+
+
+def test_project_path_not_found(tmp_path, capsys):
+    """A project_path that does not exist returns path_not_found."""
+    # Arrange
+    _make_multi_project(tmp_path)
+
+    # Act
+    rc, data = _run(capsys, tmp_path, project_path=':missing')
```

Ruff (0.15.18)

[warning] Unpacked variable `rc` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)